oid_validation/pulp/oid_validation/oid_validation.py

```diff
@@ -194,16 +194,17 @@ def _check_extensions(self, cert_pem, dest, log_func, repo_url_prefixes):
 
         valid = False
         for prefix in repo_url_prefixes:
-            # Extract the repo portion of the URL
-            repo_dest = dest[dest.find(prefix) + len(prefix):]
-            try:
-                valid = cert.check_path(repo_dest)
-            except AttributeError:
-                # not an entitlement certificate, so no entitlements
-                log_func('The provided client certificate is not an entitlement certificate.\n')
-            # if we have a valid url check, no need to continue
-            if valid:
-                break
+            if dest.find(prefix) > -1:
```

Comment:
I know you're only fixing this, but I think this whole block would be much more readable if ``os.path.commonprefix`` was used. Something like:
```
for prefix in repo_url_prefixes:
    if os.path.commonprefix([prefix, dest]) == prefix: 
        repo_dest = os.path.relpath(dest, prefix)
        repo_dest = '/' + repo_dest  # Looking at the current code I _think_ repo_dest starts with a /
    try:
        valid = cert.check_path(repo_dest)
....
```